Workbooks/Hyperscale Citus/Metrics Overview/Metrics Overview.workbook

```diff
@@ -0,0 +1,469 @@
+              "style": "info"
+            },
+            "name": "text - 1"
+          }
+        ]
+      },
+      "name": "group - 12"
+    },
+    {
+      "type": 10,
+      "content": {
+        "chartId": "workbook04f63602-6866-498b-ac24-55066d948cbb",
+        "version": "MetricsItem/2.0",
+        "size": 1,
+        "chartType": 2,
+        "resourceType": "microsoft.dbforpostgresql/servergroupsv2",
+        "metricScope": 0,
+        "resourceIds": [
+          "{Resource}"
+        ],
+        "timeContextFromParameter": "TimeRange",
+        "timeContext": {
+          "durationMs": 604800000
+        },
+        "metrics": [
+          {
+            "namespace": "microsoft.dbforpostgresql/servergroupsv2",
+            "metric": "microsoft.dbforpostgresql/servergroupsv2-Saturation-storage_percent",
+            "aggregation": 3,
+            "splitBy": "ServerName",
+            "columnName": ""
+          }
+        ],
+        "title": "Storage Percent - Max",
+        "showOpenInMe": true,
+        "showCreateAlertRule": true,
+        "gridSettings": {
+          "rowLimit": 10000
+        }
+      },
+      "name": "metric - 16",
+      "styleSettings": {
+        "showBorder": true
+      }
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "We recommend either [**scaling up**](https://go.microsoft.com/fwlink/?linkid=2187075) storage  or [**scaling out**](https://go.microsoft.com/fwlink/?linkid=2187158) the nodes if the Storage Percent exceeds 85 percent.\r\n\r\n",
+        "style": "upsell"
+      },
+      "name": "text - 12"
+    },
+    {
+      "type": 10,
+      "content": {
+        "chartId": "workbook2cf228e3-9930-4555-bb7d-98b73fb11d88",
+        "version": "MetricsItem/2.0",
+        "size": 1,
+        "chartType": 2,
+        "resourceType": "microsoft.dbforpostgresql/servergroupsv2",
+        "metricScope": 0,
+        "resourceIds": [
+          "{Resource}"
+        ],
+        "timeContextFromParameter": "TimeRange",
+        "timeContext": {
+          "durationMs": 604800000
+        },
+        "metrics": [
+          {
+            "namespace": "microsoft.dbforpostgresql/servergroupsv2",
+            "metric": "microsoft.dbforpostgresql/servergroupsv2-Saturation-storage_used",
+            "aggregation": 4,
+            "splitBy": "ServerName"
+          }
+        ],
+        "title": "Storage Used - Avg",
+        "showOpenInMe": true,
+        "showCreateAlertRule": true,
+        "gridSettings": {
+          "rowLimit": 10000
+        }
+      },
+      "name": "metric - 14",
+      "styleSettings": {
+        "showBorder": true
+      }
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "## Performance Metrics\r\n[**Click here**](https://go.microsoft.com/fwlink/?linkid=2186984) to learn more about useful diagnostic queries which might help troubleshoot performance issues.",
```

Comment:
nit: FYI, accessibility reviews would complain that "click here" isn't a useful link.  makes more sense to make the "active" part of the sentence "learn more about useful diagnostic queries" as the actual link text